src/Frontend.hs

```diff
@@ -208,6 +211,8 @@ type AulaTesting =
   :<|> "topics" :> GetH (Frame (PageShow [Topic]))
   :<|> "users"  :> GetH (Frame (PageShow [User]))
 
+  :<|> "user" :> "settings" :> FormH HTML (Html ()) ST
+
```

Comment:
Given the new state of AulaTesting, I would avoid adding handlers we have elsewhere.